repo2module/repo2module/cli.py

```diff
@@ -5,15 +5,13 @@
 import gi
 import logging
 import os
-import stat
-import sys
 
 
-gi.require_version('Modulemd', '2.0')  # noqa
-from gi.repository import Modulemd
+gi.require_version('Modulemd', '2.0')
+from gi.repository import Modulemd  # noqa: E402
 
-from dnf.subject import Subject
-import hawkey
+from dnf.subject import Subject  # noqa: E402
+import hawkey  # noqa: E402
```

Comment:
`E402` is `Module level import not at top of file`. Can you please move those imports before the `gi` block? That should fix it instead of hiding the error.

Reply:
Sure, I will move it to the top. I was not quite sure if the import order here has/hasn't some desired side effects..